src/js/base/ScrollList.tsx

```diff
@@ -5,17 +5,29 @@ import styled from "styled-components";
 import { usePrevious } from "./hooks";
 import { LoadingPlaceholder } from "./LoadingPlaceholder";
 
-function getScrollRatio(): number {
+function getScrollRatio(scrollListElement?: HTMLElement | Window): number {
+    if (scrollListElement !== window) {
```

Comment:
```suggestion
function getScrollRatio(scrollListElement?: HTMLElement ): number {
    if (boolean(scrollListElement)) {
```
Unless there is a case where you expect to pass window